src/extras/curves/EllipseCurve.js

```diff
@@ -24,23 +24,41 @@ THREE.EllipseCurve.prototype.constructor = THREE.EllipseCurve;
 
 THREE.EllipseCurve.prototype.getPoint = function ( t ) {
 
-	var deltaAngle = this.aEndAngle - this.aStartAngle;
+	// Calculate deltaAngle just once.
+	var twoPi = Math.PI * 2;
+	if ( this.deltaAngle === undefined ) {
 
-	if ( deltaAngle < 0 ) deltaAngle += Math.PI * 2;
-	if ( deltaAngle > Math.PI * 2 ) deltaAngle -= Math.PI * 2;
+		var deltaAngle = this.aEndAngle - this.aStartAngle;
+		var samePoints = Math.abs( deltaAngle ) < Number.EPSILON;
```

Comment:
I'm checking for same point for two reasons:
- when `aStartAngle `and `aEndAngle` are the same (really the same, not 2 PI related) both clockwise and counter clockwise should draw a point (in my opinion).
- when `aStartAngle` and `aEndAngle` are different but the same modulo 2 PI, both clockwise and counter clockwise should draw a full ellipse (still in my opinion).

Without the test, as in your PR, when clockwise draw a full ellipse => reverse clockwise draw a point and when clockwise draw a point => reverse clockwise draw a full ellipse. But the result is not driven by the fact that `aStartAngle` and `aEndAngle` are really the same.

The use of `Number.EPSILON` was not in my initial code, but as `Path.getPoints` uses it to know if two points are the same, I used it too.